sageresearch-app-sdk/src/main/java/org/sagebionetworks/research/sageresearch/profile/ProfileSettingsRecyclerViewAdapter.kt

```diff
@@ -193,10 +203,32 @@ class ReportProfileItemRow(val reportProfileDataItem: ReportProfileDataItem,
                            profileItem: ProfileItemProfileTableItem,
                            profileDataLoader: ProfileDataLoader) : ProfileItemRow(profileItem, profileDataLoader) {
 
+    override val detail: String?
+        get() {
+            var value = profileDataLoader.getValueString(profileItem.profileItemKey)
```

Comment:
Is there a `profileDataLoader.getValueBool()` that would standardize for different representations of a `bool`.  If not, then let's be a little more forgiving in the comparison of `value == "true"`